CSS/index.css

```diff
@@ -0,0 +1,99 @@
+body {
+    margin: 0;
+    background-color: rgb(50, 50, 50);
+    color: white;
+    font-family: "Playfair Display", Verdana, Geneva, Tahoma, sans-serif;
+}
+
+h1 {
+    text-align: center;
+    color: #eb8d46;
+}
+
+h2 {
+    color: #eb8d46;
+}
+
+main {
+    padding: 120px 20px;
+}
+
+a {
+    color: #8c98bc;
+    font-style: italic;
+}
+
+.boldFont {
+    font-weight: bolder;
+}
+
+nav {
+    display: flex;
+    gap: 30px;
+    justify-content: flex-end;
```

Comment:
Note with flex end on large displays everything shifts to right.  I assume that is what you wanted though.

Reply:
That is correct. I did want it to the right side of the page :)